test/CadlRanchProjects/inheritance/main.cadl

```diff
@@ -0,0 +1,287 @@
+import "@cadl-lang/rest";
```

Comment:
Why do we write cadl file for cadl-ranch project, we should use the one in test server?

Reply:
There are some benefits to put the test project here. I'll try to update the generate script to see if it can be automatically updated.

Reply:
done